hasher-matcher-actioner/webapp/src/pages/Matches.jsx

```diff
@@ -48,48 +50,60 @@ export default function Matches() {
   });
 
   return (
-    <div className="d-flex flex-column justify-content-start align-items-stretch h-100 w-100">
-      <div className="flex-grow-0">
-        <Container className="bg-dark text-light" fluid>
-          <Row className="d-flex flex-row justify-content-between align-items-end">
-            <div className="px-4 py-2">
-              <h1>Matches</h1>
-              <p>View content that has been flagged by the HMA system.</p>
-            </div>
-            <div className="px-4 py-2">
+    <FullWidthLocalScrollingLeftAlignedLayout title="Matches">
+      <Container className="h-100 v-100" fluid>
+        {/* ^ This container is everything below the header */}
+        <Row className="d-flex align-items-stretch h-100">
+          {/* Each child of this row spans half the available space dividing into left and right pane. */}
+          <Col
+            md={6}
+            className="d-flex flex-column justify-content-start h-100 px-0"
+            style={{borderRight: SeparatorBorder}}>
```

Comment:
@eriksendc and I had a back and forth about in code style awhile ago, Brain did we determine a standard practice we wanted to follow or no?

Reply:
Would be happy if you or @eriksendc pointed me to a place where we are doing the current standard way of doing things. 

Consider this answering all questions about `SeparatorBorder`: My understanding was that we want to stay as close to bootstrap defaults as possible so our changes would be 'local' only. Which is why inline-styles made most sense. But I'm open to other solves for this.

Reply:
You can still override without it being inline, right? Seems like if we're going to have a standard separator boarder then we'd want to centralize this... It's just not what I've seen, this constructing of an inline stile with a constant. What about having a class and then using it somewhere?